models/articles.model.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs/promises")
+const db = require("../db/connection")
+const { response } = require("../app")
+
+const fetchArticlesById = (articleId) => {
+    if(!Number(articleId)){
+        return Promise.reject({ status: 400, msg: "bad request"})
```

Comment:
This line of code is working and is achieving what it is intended to, however SQL would reject this promise for us if we allowed it to